apis/management.cattle.io/v3/rke_types.go

```diff
@@ -421,6 +421,10 @@ type IngressConfig struct {
 	ExtraVolumes []ExtraVolume `yaml:"extra_volumes" json:"extraVolumes,omitempty" norman:"type=array[json]"`
 	// Extra volume mounts
 	ExtraVolumeMounts []ExtraVolumeMount `yaml:"extra_volume_mounts" json:"extraVolumeMounts,omitempty" norman:"type=array[json]"`
+	// Http port for ingress controller daemonset
+	HttpPort string `yaml:"http_port" json:"http_port,omitempty"`
```

Comment:
I think we should set the current values 80 and 443 as default values for these two fields either here or in rke

Reply:
I have set up default values in the ingress template in kdm

Reply:
Not applicable anymore since the types have moved to RKE and I have made the changes after re-basing from upstream.